### ISSUE TITLE
feature request:  cut-and-paste blocks of layers

### ISSUE THREAD

**Issue by fugata** [xournalpp/xournalpp#2116]
tl;dr:   It would be great to be able to move -- either using "section" or "vertical space" --  multiple layers on a given page **simultaneously while preserving their relative spacing**.

Explanation:  I use xournalpp to make powerpoint like slides for my classes.   I use the layer feature to reveal blocks of texts/figures one at a time.  Many times I want to reuse chuck of these layers from an earlier slide.  Other times I realize belatedly that I need to insert/delete paragraphs in the middle of a page.  Normally I would use "vertical spacing" to slide these layers one at a time, but that could take a long time if I need to move a dozen layers.   In addition, I need need to move layers say from page 3 to page 6, if I'm not careful I could end up messing materials in the in-between pages.

It would be really great if there's an option (say in the print menu) where I can enter say layer 12-16, 18 on page 3 and move the whole block -- preserving the relative spacing -- to say 6 starting at a certain point.

I realize that using xournal(pp) as an alternative form of beamer/powerpoint is probably an "off-label" use.  That said, it's been tremendously useful over the year and I really appreciate the developers for make this tool available.  Thank you for your consideration.

**Comment by rolandlo** [comment]
duplicate of #1911
